tests/Feature/SampleFeatureTest.php

```diff
@@ -2,23 +2,30 @@
 
 namespace Saritasa\LaravelTestbed\Tests\Feature;
 
+use Illuminate\Database\Eloquent\Collection;
 use Mockery;
+use PHPUnit\Framework\AssertionFailedError;
+use Saritasa\LaravelTestbed\Traits\ApiListSortingCheck;
 use TestApp\Services\MyService;
 
 class SampleFeatureTest extends TestCase
 {
+    use ApiListSortingCheck;
+
+    /** Test to check endpoint works */
     public function testHttpRequest()
```

Comment:
You can either remove this method and `testWithMock()` or remain them in a "SampleFeatureTest" class.

Reply:
done.